src/Common/Recognizer/TimeSurveyBotLuisEx.cs

```diff
@@ -69,18 +69,86 @@ public double WorkedDurationInMinutes(string culture = Culture.English)
             }
         }
 
-        public (DateTime start, DateTime end) WorkedPeriod(double minutes, string culture = Culture.English)
+        public (DateTime start, DateTime end) WorkedPeriod(IDateTimeProvider dateTimeProvider, double minutes, TimeZoneInfo timeZone,
+            string culture = Culture.English)
         {
             var workedPeriodInstances = Entities._instance.datetime;
+            var userNow = TimeZoneInfo.ConvertTime(dateTimeProvider.DateTimeUtcNow(), timeZone);
             if (workedPeriodInstances.Length > 1)
             {
                 string? instance = workedPeriodInstances[1].Text;
-                var recognizedDateTime = DateTimeRecognizer.RecognizeDateTime(instance, culture).First();
+                var recognizedDateTime =
+                    DateTimeRecognizer.RecognizeDateTime(instance, culture, refTime: userNow).First();
                 var resolvedPeriod = ((List<Dictionary<string, string>>)recognizedDateTime.Resolution["values"])[0];
-                // TODO: use resolvedPeriod to pick a (start, end) period
```

Comment:
I think this comment refers to issue #102, maybe we should keep it

Reply:
I merged that branch into this one, that's why it was removed